ansible/roles/node_exporter_nginx/defaults/main.yml

```diff
@@ -0,0 +1,9 @@
+prometheus_basic_auth_user: "prometheus"
+prometheus_basic_auth_password: "{{ vault_prometheus_basic_auth_password }}"
+node_exporter_user: "www-data"
+node_exporter_group: "www-data"
+node_exporter_private_path: "/srv/node_exporter/private"
+node_exporter_ssl_key_path: "{{ node_exporter_private_path }}/{{ inventory_hostname }}.pem"
+node_exporter_ssl_csr_path: "{{ node_exporter_private_path }}/{{ inventory_hostname }}.csr"
+node_exporter_ssl_cert_path: "{{ node_exporter_private_path }}/{{ inventory_hostname }}.cert"
```

Comment:
IIRC, `nginx` (re)loads all ssl secrets before dropping `root` privileges. Is it the only reason to use `www-data` for certs?